src/common/transformations/src/transformations/common_optimizations/moc_transformations.cpp

```diff
@@ -244,6 +245,7 @@ bool ov::pass::MOCTransformations::run_on_model(const std::shared_ptr<ngraph::Fu
     REGISTER_PASS(manager, ReverseInputChannelsFusion)
     REGISTER_PASS(manager, AlignEltwiseInputRanks)
     REGISTER_PASS(manager, ConstantFolding)
+    REGISTER_PASS(manager, ResolveNameCollisions)
     manager.run_passes(f);
```

Comment:
ResolveNameCollisions is used to resolve friendly name conflict to ov::Model. Do I understand correctly?

Do we need a collision resolve to compiled model as well and make sure one pass would change the original name of Parameter and Result node and ports? @ilyachur @riverlijunjie @ilya-lavrenov

Reply:
do we need to care about parameter / result during model compilation?
I suppose we mostly rely on tensor names

Reply:
nncf does not work with the compiled model so it is not needed to resolve there now

Reply:
Discussed in MS Teams.
These changes affect friendly_names only. API 2.0 uses the tensor_names, so I believe we don't need to it for compiled model